XamarinCommunityToolkitSample/Resx/AppResources.Designer.cs

```diff
@@ -10,35 +10,48 @@
 
 namespace Xamarin.CommunityToolkit.Sample.Resx {
```

Comment:
Wow, what's happened here?

Reply:
Added localization to for the sample. Guess VS generates this file differently than the Mac counterpart? 🤷‍♂️

Reply:
But there are some additional auto generated changes I just saw that are not what they should be (csproj). I will go over them once more by hand and give VS an evil stare 😉